core/standalone/build.gradle

```diff
@@ -157,17 +157,19 @@ gitProperties {
 install.dependsOn(bootJar)
 
 dependencies {
-    compile "org.scala-lang:scala-library:${gradle.scala.version}"
-    compile project(':core:controller')
-    compile project(':tools:admin')
-    compile "org.rogach:scallop_${gradle.scala.depVersion}:3.3.2"
-
-    compile "io.github.embeddedkafka:embedded-kafka_${gradle.scala.depVersion}:2.4.0"
-    compile "org.scala-lang:scala-reflect:${gradle.scala.version}"
-    compile "ch.megard:akka-http-cors_${gradle.scala.depVersion}:0.4.2"
-
-    testCompile "junit:junit:4.11"
-    testCompile "org.scalatest:scalatest_${gradle.scala.depVersion}:3.0.8"
+    implementation "org.scala-lang:scala-library:${gradle.scala.version}"
+    implementation project(':common:scala')
```

Comment:
So we need to explicitly add all dependencies with `implementation`?

Reply:
Yes, The `compile` keyword is deprecated. you need to use the `implementation`.

> The usage of the compile and runtime configurations in the Java ecosystem plugins has been discouraged since [Gradle 3.4](https://docs.gradle.org/3.4/release-notes.html#the-java-library-plugin).

https://docs.gradle.org/current/userguide/upgrading_version_5.html#dependencies_should_no_longer_be_declared_using_the_compile_and_runtime_configurations

Reply:
I meant that we previously didn't need to import `common` but now we have to import `common`(and all dependencies) explicitly as we are using `implementation`.

Reply:
I don't know how it worked before. I remember that it was added because a dependency problem occurred.